### PR TITLE
Cosbench

### PR DESCRIPTION
**Pull request by fatpat**
this generate the package `cosbench` that contains the common files for both `controller` and `drivers`.

A sub package `cosbench-controller` for the controller only

and a sub package `cosbench-driver` yet to come

### CODE REVIEW

File: cosbench/cosbench.spec [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+%define        cosbench_home    %{_sharedstatedir}/cosbench
+%define        cosbench_prefix  %{_prefix}/cosbench
+%define        cosbench_etc     %{_sysconfdir}/cosbench
+%define        cosbench_user    cosbench
+%define        cosbench_group   %{cosbench_user}
+
+%package controller
+Summary:       Cosbench Controller
+Requires:      jre6
```
Comment (cdelgehier) [inline cosbench/cosbench.spec]: `jre6` ? Are you sure of you ? `java-1.8.0-openjdk-headless` seems to be better

Comment (fatpat) [reply cosbench/cosbench.spec]: cosbench is compiled with java1.6 and requires it at compilation: `Bundle-RequiredExecutionEnvironment: JavaSE-1.6`.

it works with newer version for sure. I don't know what's best ... real question

Comment (fatpat) [reply cosbench/cosbench.spec]: @fvennetier, what do you think ?

Comment (fvennetier) [reply cosbench/cosbench.spec]: jre6 has been deprecated for a while. I would use the more recent version that works, and I think it is java-1.8.0-openjdk.

**Comment by fatpat** [comment]
Ready to merge, @vincent-legoll, last word before ignition ? ;)

**Pull request merged**